apps/backend/test/users.e2e-spec.ts

```diff
@@ -594,6 +583,15 @@ describe('/users', () => {
           });
       });
 
+      it('should not be able to update themselves without providing the current password', async () => {
+        return update(
+          app,
+          editUserId,
+          UPDATE_USER_DTO_WITH_MISSING_CURRENT_PASSWORD_FIELD,
+          jwtToken
+        ).expect(HttpStatus.OK);
```

Comment:
Why return OK when the test description/title is written like its expecting a failure?